gel/_internal/_qbmodel/_abstract/_primitive.py

```diff
@@ -349,6 +351,31 @@ def _get_py_type_for_abstract_scalar(typename: str) -> set[tuple[str, str]]:
     return union
 
 
+MODEL_SUBSTRATE_MODULE: Final = "__gel_substrate__"
```

Comment:
would love to have a comment here explaining what substrate means and  what the attribute is supposed to do.

Reply:
added